networkx/algorithms/bipartite/matching.py

```diff
@@ -183,14 +184,18 @@ def eppstein_matching(G):
     hopcroft_karp_matching
 
     """
+    # Due to its original implementation, a directed graph is needed
+    # so that the two sets of bipartite nodes can be distinguished
+    Di_G = nx.DiGraph()
```

Comment:
You should probably just overwrite the variable `G` here instead of creating a new variable, that way you don't have to change the variable names in the lines below.